libraries/AC_PID/AC_P_1D.h

```diff
@@ -17,22 +17,29 @@ class AC_P_1D {
     // set time step in seconds
     void set_dt(float dt) { _dt = dt; }
 
+    // get maximum error
+    float get_error_min() const {return _error_min;}
```

Comment:
nitpick: we should have a space after/before the curly brackets.  same thing on the line below.